seaborn/tests/test_utils.py

```diff
@@ -281,3 +289,30 @@ def test_category_order():
 
         out = utils.categorical_order(x, ["b", "a"])
         nt.assert_equal(out, ["b", "a"])
+
+
+@network(url="https://github.com/mwaskom/seaborn-data")
```

Comment:
Yeap... that is its purpose + also tag the test with network tag ask you could skip it altogether (on the phone atm, so man nosetests for arg, smth like -X "not network")

On April 28, 2015 2:38:20 PM EDT, Michael Waskom <notifications@github.com> wrote:
>> @@ -281,3 +289,30 @@ def test_category_order():
>>  
>>          out = utils.categorical_order(x, ["b", "a"])
>>          nt.assert_equal(out, ["b", "a"])
>> +
>> +
>> +@network(url="https://github.com/mwaskom/seaborn-data")
>
>Does this decorator allow the test to get skipped when there's no
>network connection?
>
>---
>Reply to this email directly or view it on GitHub:
>https://github.com/mwaskom/seaborn/pull/526/files#r29274124